src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2Request.java

```diff
@@ -187,6 +197,11 @@ private void errorResult(Throwable e, HTTP2SampleResult res) {
     protected void sample(URL url, String method, HTTP2Connection http2Connection,
                           HTTP2SampleResult sampleResult) {
 
+        String urlStr = url.toString();
+        if (log.isDebugEnabled()) {
+            log.debug("Start : sample: " + urlStr + ", method: " + method);

```

Comment:
Using what it was recommended in last [comment](https://github.com/Blazemeter/jmeter-http2-plugin/pull/7/files#r369030479) (usage of slf4j), it won't be needed to concatenate the log message. It could be something like `log.debug("Start:  sample: {}, method: {}, url.toString(), method")`